src/stateManagement/stackSpecificStateManager.js

```diff
@@ -11,11 +11,11 @@ import {
  * @constructor newStackSpecificToolStateManager
  * @memberof StateManagement
  *
- * @param  {string[]} toolTypes       The tool types to apply to the stack.
+ * @param  {string[]} toolNames       The tool names to apply to the stack.
```

Comment:
This description is odd, I had to double check usage, but `toolNames` does appear to be what we want here.

Reply:
Updated the jsDoc giving a better explanation of this parameter.